### ISSUE TITLE
Not all messages are recovered

### ISSUE THREAD

**Issue by Sunkwii** [ItzDerock/discord-html-transcripts#35]
I just used the module for the first time and when I open the file, only 24 messages are displayed on a total of 1184.

This problem appears only with version 2.5.6 of the module.
Older versions like 2.5.5 work fine.

discord.js: v13.7.0

**Comment by ItzDerock** [comment]
Can you provide any options you are passing to the function?

**Comment by ItzDerock** [comment]
Unable to reproduce, exported ~3k messages fine with `returnType: 'buffer'` as the only option.

**Comment by ItzDerock** [comment]
exported ~19k messages and still works fine

**Comment by Sunkwii** [comment]
```js
  const channel = discord.channels.cache.get('963640501028929556');

  const attachment = await discordTranscripts.createTranscript(channel, {
    limit: -1,
    returnType: 'attachment',
    fileName: 'transcript.html',
    minify: false,
    saveImages: false,
    useCDN: true
  });

channel.send({
      files: [attachment]
    });
```

**Comment by Sunkwii** [comment]
I tried with your options, I only get `file.jpg` which is a simple white image

**Comment by ItzDerock** [comment]
> I tried with your options, I only get `file.jpg` which is a simple white image

That's because I set `returnType` to buffer. 
I set it to that so I could save it as a file directly instead of sending to discord. 
this shouldn't affect the transcript though

**Comment by ItzDerock** [comment]
and I think I just found the issue, pushing a fix soon

**Comment by Sunkwii** [comment]
Ok, thank you very much

**Comment by ItzDerock** [comment]
released as version `2.5.7`
